src/components/Router.tsx

```diff
@@ -0,0 +1,15 @@
+import { Navigate, Route, Routes } from 'react-router-dom';
+import { HomePage } from '../pages/HomePage/HomePage';
+import { PeoplePage } from '../pages/PeoplePage/PeoplePage';
+
+export const Router: React.FC = () => (
+  <Routes>
+    <Route path="*" element={<h1 className="title">Page not found</h1>} />
```

Comment:
Would be better if you create a separate page for this
```suggestion
    <Route path="*" element={<NotFoundPage />} />
```